tests/test_octodns_provider_cloudflare.py

```diff
@@ -599,7 +599,7 @@ def test_cdn(self):
         zone = Zone('unit.tests.', [])
         provider.populate(zone)
 
-        # the two A records get merged into one CNAME record poining to the CDN
+        # the two A records get merged into one CNAME record pointing to the CDN
```

Comment:
`tests/test_octodns_provider_cloudflare.py:602:80: E501 line too long (80 > 79 characters)`
Error causing the test failure